Compass/AccountSettingsViewController.swift

```diff
@@ -0,0 +1,246 @@
+        
+        NSLayoutConstraint.activate([avatarImage.heightAnchor.constraint(equalToConstant: 80),
+                                     avatarImage.widthAnchor.constraint(equalToConstant: 80),
+                                    avatarImage.leftAnchor.constraint(equalTo: userimageView.leftAnchor),
+                                    avatarImage.bottomAnchor.constraint(equalTo: userimageView.bottomAnchor)])
+
+        NSLayoutConstraint.activate([updateUserImageButton.heightAnchor.constraint(equalToConstant: 20),
+                                     updateUserImageButton.widthAnchor.constraint(equalToConstant: 20),
+                                     updateUserImageButton.rightAnchor.constraint(equalTo: userimageView.rightAnchor),
+                                     updateUserImageButton.bottomAnchor.constraint(equalTo: userimageView.bottomAnchor)])
+        
+        NSLayoutConstraint.activate([userInforTextField.heightAnchor.constraint(equalToConstant: 200)])
+    }
+    
+    private func setUpHandlers() {
+        deleteAccountButton.addTarget(self, action: #selector(deleteAccount), for: .touchUpInside)
+        updateUserImageButton.addTarget(self, action: #selector(updateUserImage), for: .touchUpInside)
+    }
+    
+    @objc private func deleteAccount() {
+        
+        let alert = UIAlertController(title: "Account will be deleted", message: "Do you wish to continue?", preferredStyle: UIAlertControllerStyle.alert)
+        alert.addAction(UIAlertAction.init(title: "Yes", style: .default, handler: { (action) in
+            
+            self.activityIndicator = UIActivityIndicatorView(frame: CGRect.zero)
+            self.activityIndicator.center = self.view.center
+            self.activityIndicator.hidesWhenStopped = true
+            self.activityIndicator.activityIndicatorViewStyle = UIActivityIndicatorViewStyle.gray
+            self.view.addSubview(self.activityIndicator)
+            self.activityIndicator.startAnimating()
+            UIApplication.shared.beginIgnoringInteractionEvents()
+            
+            self.par.deleteUser(completition: { (success, error) in
+                guard error == nil else {
+                    var displayErrorMessage = "Error while deleting account, please try again"
+                    let error = error as NSError?
+                    if let errorMessage = error?.userInfo["error"] as? String {
+                        displayErrorMessage = errorMessage
+                        
+                    }
+                    self.createAlert(title: "Error:", message: displayErrorMessage)
+                    return
+                }
+                
+                if success {
+                self.activityIndicator.stopAnimating()
+                UIApplication.shared.endIgnoringInteractionEvents()
+                self.navigationController?.popToRootViewController(animated: true)
```

Comment:
Minor issue, I'd fix the indentation here so that it's:
```
if success {
    self.activityIndicator.stopAnimating()
    UIApplication.shared.endIgnoringInteractionEvents()
    self.navigationController?.popToRootViewController(animated: true)
}